config/log4j.properties

```diff
@@ -23,13 +24,23 @@ log4j.appender.streams=org.apache.log4j.ConsoleAppender
 log4j.appender.streams.layout=org.apache.log4j.PatternLayout
 log4j.appender.streams.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
-log4j.logger.kafka=ERROR, stdout
+# loggers
+
+log4j.logger.org.reflections=WARN, stdout
```

Comment:
can we keep this one at ERROR?  Connect spams the logs with a ton of messages without this one when it loads the plugins. I think that's the only place it's used

Reply:
Sure.